### DISCUSSION TITLE
Test FedCM for BYOIDP

### DISCUSSION THREAD

**Discussion by erlend-sh** [authelia/authelia#7207] [Ideas]
For advocates of digital identity ownership, right now there’s a unique opportunity to tip the scales in favor of the independent web:

https://wrily.foad.me.uk/sign-in-with-big-tech-only-or-sign-in-with-whom-i-prefer

The FedCM drafters have been open to reference implementations for a month: https://github.com/fedidcg/FedCM/issues/240#issuecomment-2004650817

Apparently there’s not even a need to use Chrome Canary for testing any longer, as an experimental option for FedCM has been added to main Chrome:

<img width="819" alt="image" src="https://github.com/portier/portier-broker/assets/583842/d2f1a3a7-4c00-4d91-9f4b-d16d678e629c">

So far there’s one nearly [working example in Rauthy](https://github.com/sebadob/rauthy/discussions/145#discussioncomment-8831943).

With a few more proofs-of-concept, we can ensure that this emerging browser standard will benefit the entirety of the open web. Your assistance would be hugely appreciated by your fellow indies! 🙏

**Comment by glime-ai** [comment]
Please note this response was created using generative AI leveraging previously available information from the repository. This information may be helpful however it is our strong recommendation that you carefully consider the information before blindly applying it.

---

It seems like you are discussing the potential for implementing the FedCM (Federated Credential Management) API in a project or advocating for its adoption to enhance digital identity management on the independent web. However, your message does not contain a clear question or request related to Authelia or its capabilities directly.

Could you please clarify if you are seeking information on Authelia's current or planned support for FedCM, or if you need assistance with something specific related to Authelia's authentication features? This will help me provide the most accurate and helpful information.

This comment was generated by [Glime](https://glimelab.ai/?utm_source=6fsHkcyufjQLaB&utm_medium=github).

**Comment by james-d-elliott** [comment]
Thanks for the idea and we'll keep it in mind as it and Authelia evolves, especially as we dive into the areas that it has the most impact (session management / front channel logout).
